src/Menu/MenuBuilder.php

```diff
@@ -23,11 +23,12 @@ public function __construct(
     public function createMainMenu(): ItemInterface
     {
         $menu = $this->factory->createItem('root');
+
         $menu->setChildrenAttribute('class', 'nav navbar-nav');
 
         $this->eventDispatcher->dispatch(
             new ConfigureMenuEvent(
-                $this->getFactory(),
```

Comment:
I think the method should be removed also, but at first glance there is no method getFactory()

Reply:
The diff for this file isn't complete, the method was removed (https://github.com/sumocoders/FrameworkCoreBundle/pull/65/files#diff-fd2598d5acbaf7658294998afc5cf2873a32d946e2d03875c8288ef5f0583e32L49)